new-dti-website/components/bottom.tsx

```diff
@@ -35,9 +35,9 @@ const Bottom: React.FC = () => (
       <div className="text-left w-full md:self-center max-w-[520px] relative z-10">
         <div className="flex flex-col gap-y-1 pb-3 md:gap-y-2">
           <p className="text-neutral-200 text-xs font-bold uppercase lg:text-lg">courses</p>
-          <h5 className="text-lg font-semibold md:text-xl lg:text-[32px]">
+          <h3 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```

Comment:
```suggestion
          <h2 className="text-lg font-semibold md:text-xl lg:text-[32px]">
```


wait wdyt of making these ones `<h2>`


i'm looking at the homepage and it seems like these headers have the 2nd biggest font size on the page?
<img width="1599" alt="Screenshot 2024-12-01 at 12 32 52 PM" src="https://github.com/user-attachments/assets/8c3f0b68-d823-47c7-8ebe-4884b648e455">